docker/tomcat_expedius/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM tomcat:7
+
+# Fetch a modified tomcat-users and server config.
+ADD ./conf/tomcat-users.xml /usr/local/tomcat/conf/tomcat-users.xml
+
+# Modified server.xml is set with a 8444 port 
+ADD ./conf/server.xml /usr/local/tomcat/conf/server.xml
+
+# Deploy the tomcat war file.
+ADD Expedius*.war /usr/local/tomcat/webapps/Expedius.war
```

Comment:
Where do I download this from in order to build?

Reply:
Interesting, doesn't the war file stay inside the tomcat_expedius image?  I will set up a private repository to connect with right now.